MANUAL.txt

```diff
@@ -2142,6 +2142,56 @@ ODT or pptx.
 [Unicode Bidirectional Algorithm]: https://www.w3.org/International/articles/inline-bidi-markup/uba-basics
 [Language subtag lookup]: https://r12a.github.io/app-subtags/
 
+### Variables for HTML
+
+`document-css`
+:   Enables inclusion of most of the CSS in the `styles.html`
+    [partial](#partials) (have a look with
+    `pandoc --print-default-data-file=templates/styles.html`).
+    This variable is set to `true` by default, unless you use
+    [`--css`](#option--css).
```

Comment:
It occurs to me that this would be a breaking change -- currently, the bits of styling inserted by pandoc through styles partial are included even if you specify `--css`, though you can override them using `--css`.  Is there a reason not to keep this behavior?

Reply:
The bits that were in the partial already before this change are not part of the `$if(document-css)$` block.. so they'll still always be included.